pkg/fdbstatus/status_checks_test.go

```diff
@@ -842,7 +842,7 @@ var _ = Describe("status_checks", func() {
 					},
 				},
 			}),
-		Entry("when recovered since is enabled and version supports it",
+		Entry("when recovered since is enabled and version supports it it should use the minimum of recovered since or process uptime",
```

Comment:
```suggestion
		Entry("when recovered since is enabled and version supports it, it should use the minimum of recovered since or process uptime",
```